app/views/candidate_interface/work_history/review/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, page_title('work_history') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
```

Comment:
maybe locale for `Back to application`

Reply:
![Screenshot 2019-10-31 at 14 47 34](https://user-images.githubusercontent.com/1650875/67957357-61508a80-fbed-11e9-9dde-95f9dabd6f30.png)

Ehhh it's fine 🤣